src/Frontend/Page/Idea.hs

```diff
@@ -104,6 +107,13 @@ data ReportComment = ReportComment Comment
 
 instance Page ReportComment where
 
+data EditComment
+    = EditComment Idea Comment
```

Comment:
just learned that `Idea` is needed for its title.  In that case we should still simplify this type and only carry around the title.  Less confusing.